### ISSUE TITLE
Feature - full block history

### ISSUE THREAD

**Issue by sternschnaube** [mastercion/minecraft_mod_fabric_blockowner#8]
A history of each block, also destroyed ones.

**Comment by mastercion** [comment]
Hey, thanks for your suggestion!

This was indeed my initial goal, and it remains so. However, achieving this poses many challenges that i will need to address before the full release.

Do you have any ideas on how to display the history of each book? Since this is a server-side mod, we can't easily use custom GUIs (at least not to my knowledge).

**Comment by venaminecraft** [comment]
Polymer should be able to help you with server-side GUIs: 
https://modrinth.com/mod/polymer

Or this mod there: https://modrinth.com/mod/skiesguis

I don't think there's a need for a custom GUI. Just display the history for the last 5-10 blocks. With page support for longer block histories 😄

**Comment by venaminecraft** [comment]
Two new features:
- Being able to print the result into the chat, not as title 
- Allowing non-op players to use the mod

**Comment by mastercion** [comment]
> Two new features:
> 
> * Being able to print the result into the chat, not as title
> * Allowing non-op players to use the mod

Yeah, I will definitely add those. However, my primary intention with this mod is to enable users to see who placed a block just by looking at it.

I can make the title message optional.

>I don't think there's a need for a custom GUI. Just display the history for the last 5-10 blocks. With page support for longer block histories 😄

Yeah, this is a good way of overviewing the block history. I will use this approach.

Thanks for reaching out, I will keep your request in mind.
